scripts/install-windows-prereqs.ps1

```diff
@@ -360,6 +362,31 @@ function Install-Python3 {
     Move-Item -Path "$tempInstallDir\*" -Destination $installDir
     Add-ToSystemPath -Path $installDir
 
+    Start-ExecuteWithRetry -ScriptBlock {
+        # Install the PIP installer
+        $Destination = Join-Path $installDir "get-pip.py"
+        Start-FileDownload -URL $GetPipURL `
```

Comment:
Why not just use the existing infrastructure for this by adding the PIP package to `$PACKAGES` where it will get downloaded and hash checked as part of the existing `Start-LocalPackagesDownload` function?

Reply:
Great question, let me see if I can do that.

Reply:
The problem with that is that PIP and cmake_format have to go inside the python directory, which is not created until well after Start-LocalPackagesDownload completes, and then Install-Python3 gets called which unzips the python download into that directory.  So at most it could be downloaded and then copied around later, but that seems more work than just delaying the download until it can be downloaded in place.   (That's similar to how esy gets installed by waiting until node is installed and downloading it then so the precedent already exists.)

Reply:
Done

Reply:
I'm not sure I understand why get-pip.py needs to be copied anywhere after it's downloaded? In my experience, get-pip.py installs to the path of the python.exe instance that's executing it, so the only change required here is to add the get-pip.py entries to `$PACKAGES`, then after python is installed, call `python $PACKAGES["get-pip"]["local_file"]`.

Reply:
Already done :)